Unix/delete_user.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+username=$OMI_USER
+
+if [ "x$username" = "x" ]; then
+    username=omi_$USER
+fi
+
+egrep "^$username" /etc/passwd >/dev/null
+if [ $? -eq 0 ]; then
+    userdel=`which userdel`
+    if [ "x$userdel" = "x" ]; then
+        echo "Failed to find 'userdel'"
+        exit 1
+    fi
+    sudo $userdel --force $username
```

Comment:
Do you need a qualifier to delete the home directory? And does a group get created for the user? If so, is that cleaned up?